code/modules/martial_arts/martial.dm

```diff
@@ -6,19 +6,31 @@
 	var/streak = ""
 	var/max_streak_length = 6
 	var/temporary = FALSE
-	var/datum/martial_art/base = null // The permanent style
-	var/deflection_chance = 0 //Chance to deflect projectiles
-	var/block_chance = 0 //Chance to block melee attacks using items while on throw mode.
+	/// The permanent style
+	var/datum/martial_art/base = null
+	/// Chance to deflect projectiles while on throw mode
+	var/deflection_chance = 0
+	/// Can it reflect projectiles in a random direction?
+	var/reroute_deflection = FALSE
+	///Chance to block melee attacks using items while on throw mode.
+	var/block_chance = 0
```

Comment:
I'm not actually sure what the standard is for punctuation in comments, but some of these have full stops and some don't. Personally I'd just add it on to all of them.